tests/e2e-openshift/otlp-metrics/04-metrics-traces-gen.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: otelgen-traces
+spec:
+  completions: 1
+  parallelism: 1
+  template:
+    metadata:
+      labels:
+        app: otelgen
+    spec:
+      containers:
+        - name: otelgen-traces-multi
+          image: ghcr.io/krzko/otelgen:latest
+          env:
+            - name: OTLP_ENDPOINT
+              value: cluster-collector-collector-headless.kuttl-otlp-metrics.svc:4317
+          command: ["./otelgen"]
+          args:
+            - "--otel-exporter-otlp-endpoint=$(OTLP_ENDPOINT)"
+            - "--log-level=debug"
+            - "--insecure=true"
+            - "--duration=30"
+            - "traces"
```

Comment:
Why traces are reported? 

Could we limit the scope for reporting and asserting only metrics (e.g. get rid of Jaeger as well)?

Reply:
I recommend retaining the metrics and traces check as it enables comprehensive testing. By sending both to the collector endpoint, we can thoroughly test for any regressions and ensure a comprehensive evaluation of the system.

Reply:
In that case I would recommend renaming the test since the scope is much wider. 

For openshift tests we can either have well defined tests with a narrow scope or a single one that tests most of the functionality (it's going to be harder to figure out what is wrong, but the execution time will be better). For now I would maybe prefer to have a single test for OCP to test various features (e.g. OTLP reporting, route creation etc)

Reply:
@pavolloffay  Updated the test name.